.gitignore

```diff
@@ -28,6 +28,7 @@ msbuild.log
 msbuild.err
 msbuild.wrn
 msbuild.binlog
+msbuild.ProjectImports.zip
```

Comment:
NIT: perhaps just include here *.binlog as we probably don't want to check any of those in, and also remove these ProjectImports.zip, these only are there if you cancelled a build that was producing binlogs, so they shouldn't be here.

Reply:
*.binlog sounds good, thanks.